crates/anvil/core/src/types.rs

```diff
@@ -38,6 +38,13 @@ pub struct ReorgOptions {
     pub tx_block_pairs: Vec<(TransactionData, u64)>,
 }
 
+/// Represents the options used in `anvil_rollback`
+#[derive(Debug, Clone, Deserialize)]
+pub struct RollbackOptions {
```

Comment:
not sure this is needed, we can just have `Rollback(u64)` (or even more nicely `Rollback(Option<u64>)` which rollbacks to prev if none?)

Reply:
much cleaner, done!